scripts/Jenkinsfiles/quality

```diff
@@ -0,0 +1,219 @@
+def runQualityTests() {
```

Comment:
Are there any best practices for naming these? Perhaps we should name it with something terminating in .groovy for syntax highlighting purposes in text editors?